src/core/fonts.js

```diff
@@ -3227,7 +3227,7 @@ var Font = (function FontClosure() {
        */
       function readCmapTable(cmap, font, isSymbolicFont, hasEncoding) {
         var segment;
-        var start = (font.start ? font.start : 0) + cmap.offset;
+        var start = (font.start ? font.start : 0) + (cmap ? cmap.offset : 0);
```

Comment:
I think that that would become messy on the caller side, since we're relying on `readCmapTable` to return something in order for the glyph mapping to work (see https://github.com/mozilla/pdf.js/blob/master/src/core/fonts.js#L4224-L4229 and below).

How about the updated version of the patch, which returns immediately for this (hopefully) rare case?